src/elements/toggle/toggle-option/toggle-option.visual.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { html, nothing } from 'lit';
```

Comment:
Wondering if it is worth to have separate toggle option tests at all, or if the toggle tests are sufficient? What do you think?

Reply:
Since there are only a few cases, I don't think there's any harm in keeping them